eli5/xgboost.py

```diff
@@ -97,7 +99,10 @@ def explain_prediction_xgboost(
         target_names=None,
         targets=None,
         feature_names=None,
-        vectorized=False):
+        vectorized=False,
+        feature_re=None,
+        feature_filter=None,
```

Comment:
I don't mind any argument order, but it'd be nice to have it consistent - e.g. in `explain_prediction_tree_regressor` `vectorized` is the last argument.